src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -305,6 +305,7 @@ private String getSiteName(String siteId, String sessionId) {
 					String.class);
 			JSONObject siteJSONObject = new JSONObject(siteJson);
 			siteName = (String) siteJSONObject.get("title");
+			siteName=Utils.sanitizeName(Utils.COLLECTION_TYPE,siteName);
```

Comment:
Are there any other calls to sanitizeName?  If so does the change work for them?